src/utils/account/loginUserProgramatically.ts

```diff
@@ -0,0 +1,10 @@
+import { loginAction } from 'redux/commonActions';
+import { store } from 'redux/store';
+import { LoginMethodsEnum } from 'types';
+
+export function loginUserProgramatically(
```

Comment:
maybe change name to _loginWithExtraProvider_, to reflect what function actually does